src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import React from 'react';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage | '';
+  onClose: (message: '') => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  onClose,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: !errorMessage },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => onClose('')}
```

Comment:
```suggestion
        onClick={handleCloseNotification}
```

and add `const handleCloseNotification = () => {}` helper function outside the JSX, 
I think, @h-amster supposed exactly that